Tasks/PythonScriptV0/task.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "6392F95F-7E76-4A18-B3C7-7F078D2F7700",
+    "name": "PythonScript",
+    "friendlyName": "Python Script",
+    "description": "Run a Python script.",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 135,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "instanceNameFormat": "Run Python script",
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "targetType",
+            "type": "radio",
+            "label": "Type",
+            "required": true,
+            "defaultValue": "filePath",
+            "helpMarkDown": "Target script type: File path or Inline",
+            "options": {
+                "filePath": "File path",
+                "inline": "Inline"
+            }
+        },
+        {
+            "name": "filePath",
+            "type": "filePath",
+            "label": "Script Path",
+            "visibleRule": "targetType = filePath",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "Path of the script to execute. Must be a fully qualified path or relative to $(System.DefaultWorkingDirectory)."
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "visibleRule": "targetType = inline",
+            "required": true,
+            "defaultValue": "",
+            "properties": {
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "5000"
+            },
+            "helpMarkDown": "The Python script to run"
+        },
+        {
+            "name": "arguments",
+            "type": "string",
+            "label": "Arguments",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Arguments passed to the script execution, available through `sys.argv`."
```

Comment:
The Bash task only lets you pass arguments when you're running a script from a file.  I don't see any reason for that restriction (in Python at least), so this task lets you pass arguments to inline scripts as well.